r/accessing-and-managing-financial-data.qmd

```diff
@@ -118,40 +137,40 @@ factors_q_monthly_link <-
 
 factors_q_monthly <- read_csv(factors_q_monthly_link) |>
   mutate(date = ymd(str_c(year, month, "01", sep = "-"))) |>
-  select(-R_F, -R_MKT, -year) |>
-  rename_with(~ str_remove(., "R_")) |>
-  rename_with(~ str_to_lower(.)) |>
-  mutate(across(-date, ~ . / 100)) |>
+  rename_with(~sub("R_", "", ., fixed = TRUE)) |>
+  rename_with(tolower) |>
+  mutate(across(-date, ~. / 100)) |>
+  select(date, risk_free = f, mkt_excess = mkt, everything()) |>
   filter(date >= start_date & date <= end_date)
```

Comment:
this can be down with tidyfiannce as well

Reply:
Indeed, as it says in the lines below anyway.